models.rdf

```diff
@@ -16,21 +16,10 @@
       <dcterms:publisher rdf:resource="http://www.duraspace.org/"/>
       <rdfs:seeAlso rdf:resource="https://github.com/duraspace/pcdm/wiki"/>
       <rdfs:comment xml:lang="en">Ontology for the Portland Common Data Model, intended to underlie a wide array of repository and DAMS applications.</rdfs:comment>
-      <owl:versionInfo>2015/10/14</owl:versionInfo>
-      <owl:priorVersion rdf:resource="http://pcdm.org/2015/09/28/models"/>
+      <owl:versionInfo>2016/04/18</owl:versionInfo>
+      <owl:priorVersion rdf:resource="http://pcdm.org/2015/10/14/models"/>
```

Comment:
[This commit](https://github.com/duraspace/pcdm/commit/2c3cd6932d0a2ae8d3748729e3c769a8854a82bc) is incorrect. The `priorVersion` should have been unchanged, as:
```
<owl:versionInfo>2016/04/18</owl:versionInfo>
<owl:priorVersion rdf:resource="http://pcdm.org/2015/09/28/models"/>
```
Please rollback and update.